test/ex_machina_test.exs

```diff
@@ -85,8 +83,18 @@ defmodule ExMachinaTest do
     assert comment.article == my_article
   end
 
-  test "factorys can be defined without the second options param" do
-    assert MyApp.ExMachina.build(:without_opts) == MyApp.ExMachina.factory(:without_opts)
+  test "assoc/3 can specify a factory for the association" do
```

Comment:
I think it would be good to put this above line 78. So we test the method directly, and then at the end test that it can be used inside the module